.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Dear friend, we, the rest of the NativeScript community thank you for your
+contribution! Because we want to present a really nice, readable changelog with each
+release, please provide the following information:
+
+### Create a meaningful title
```

Comment:
Can we add a `Description` paragraph instead of this?
```
### Description
A few sentences describing the overall goals of the pull request's commits.
```
Also place the advice regarding PR titles in a comment block. __<!--- comment -->__

Have a paragraph for PRs related to this one? Like dependencies on this PR?
```

## Related Pull Requests
List related Pull requests against other repos/branches:

repo | branch | PR
------ | -------- | ------
repo | branch | [#xxx]()
```